samples/ingress/database-api/.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
             "request": "launch",
             "preLaunchTask": "build",
             // If you have changed target frameworks, make sure to update the program path.
```

Comment:
💡 I wonder comments like this have any historical background that we can keep them for those reasons?

Reply:
@Tatsinnit Yes we are keeping these, it is just that GitHub displays them in a weird way as if it is deleted.